tests/tests.c

```diff
@@ -1618,8 +1656,8 @@ bool sirtest_filesystem(void) {
 # endif
         {"/dev", true},
 #else /* __WIN__ */
-        {"\\Windows", true},
-        {"\\Program Files", true},
+        {"C:\\Windows", true},
```

Comment:
If this isn't needed for Wine or something, \ is translated to "root" or `/` `C:\` (or whatever disk Windows is installed on, which isn't guaranteed to be named C).

Reply:
Ah, see, this is failing under Wine, it was always going for the path based on the current working directory, and under wine that is by default "Z:" (`Z:` maps to Unix `/`), so it was looking for `Z:\Windows` and no good.  There is always a `C:\Windows` on Wine.

Reply:
@aremmell Not sure how you'd want to handle this one then.

Reply:
I will write a new getcwd. Gimmie 10 mins. If that one still returns Z: then we can leave it C:\

Reply:
I'll add OrangeC to the CI/CD job too, so we can see what it does in the test.